src/open_clip/transformer.py

```diff
@@ -0,0 +1,320 @@
+from collections import OrderedDict
```

Comment:
Is there any diff compared to code in main ?

Reply:
no